packages/waku/src/router/create-pages.ts

```diff
@@ -227,27 +236,26 @@ export const createPages = <
     [PathSpec, FunctionComponent<any>]
   >();
   const apiPathMap = new Map<
-    string,
+    string, // `${method} ${path}`
```

Comment:
Seems room for improvement, but just a nit.

Reply:
do you have any suggestions? this composite key approach feels a tiny bit tricky to use, but it solves the problem really well

Reply:
Not right away. I'll let you know if something arises.